README.md

```diff
@@ -134,6 +134,20 @@ resolves the path to the module that is the package's entry point:
   but returns an object with two properties:
   * `module`: the module returned by `await import(importPath)`.
   * `modulePath`: the full path to the module (file) that is the entry point to the package/module.
+  
+`quibble` allows mocking node's internal modules (e.g. "[fs](https://nodejs.org/docs/latest-v14.x/api/fs.html)") too.
```

Comment:
I would rewrite a bit to make it a note (with ">"):

> Note that when mocking internal Node.js modules (e.g. "[fs](https://nodejs.org/docs/latest-v14.x/api/fs.html)")), you need to mock the named exports both as named exports and as properties in the default export, because Node.js exports internal modules both as named exports and as a default object. Example: